websites/T/Tidal/presence.ts

```diff
@@ -33,16 +33,16 @@ presence.on("UpdateData", async () => {
 	}
 
 	const presenceData: PresenceData = {
-			largeImageKey: "logo",
-		},
+		largeImageKey: "logo",
+	},
 		songTitle = document.querySelector<HTMLAnchorElement>(
 			'div[data-test="footer-track-title"] > a'
 		),
 		currentTime = document
-			.querySelector<HTMLElement>("time.current-time")
+			.querySelector<HTMLElement>('time[data-test="current-time"]')
 			.textContent.split(":"),
 		endTime = document
-			.querySelector<HTMLElement>("time.duration-time")
+			.querySelector<HTMLElement>('time[data-test="duration"]')
```

Comment:
judging by the fact that these selectors include the word "test" they seem like they could be temporary, do you have any way of confirming that these won't change?

Reply:
They have been like this for a long time now, Tidal's upate is here to stay :)